utils.mjs

```diff
@@ -0,0 +1,23 @@
+export const titleToHtmlFriendly = (brandTitle) =>
+  brandTitle
+    .replace(/&/g, '&amp;')
+    .replace(/"/g, '&quot;')
+    .replace(/</g, '&lt;')
+    .replace(/>/g, '&gt;')
+    .replace(/./g, (char) => {
+      const charCode = char.charCodeAt(0);
+      return charCode > 127 ? `&#${charCode};` : char;
+    });
+
+export const htmlFriendlyToTitle = (htmlFriendlyTitle) =>
+  htmlFriendlyTitle
+    .replace(/&#([0-9]+);/g, (_, num) => String.fromCharCode(parseInt(num)))
+    .replace(
+      /&(quot|amp|lt|gt);/g,
+      (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' }[ref]),
+    );
+
+export const getSvg = (icon) =>
+  `<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>${titleToHtmlFriendly(
+    icon.title,
+  )}</title><path d="${icon.path}"/></svg>`;
```

Comment:
I've checked in the past what is faster: `string${interpolation}` or `'string' + addition` and the second is. Also, string interpolation is an ES6 feature, right? See [this thread](https://github.com/simple-icons/simple-icons/pull/6505/files#r706527151). I suppose that ESBuild converts to additions, but I'm not sure as you're targetting the build for NodeJS v14. Note that for the package [we are targetting all NodeJS versions as of 0.12.18](https://github.com/simple-icons/simple-icons/blob/986589c34c1224fcd0244e07a8fe833079fd836f/package.json#L76-L78), so the `engine` would be obsolete after this change and we are removing official support for a lot of legacy versions. I prefer not to do it.

Reply:
Let's target Node 14 as Node 12 is end of life